2week/src/main/java/com/pironeer/_week/controller/ComCommentController.java

```diff
@@ -0,0 +1,48 @@
+package com.pironeer._week.controller;
+
+
+
+import com.pironeer._week.dto.request.ComCommentCreateRequest;
+import com.pironeer._week.dto.request.CommentUpdateRequest;
+import com.pironeer._week.dto.response.ComCommentResponse;
+import com.pironeer._week.repository.ComCommentRepository;
+import com.pironeer._week.service.ComCommentService;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/com_comment")
+public class ComCommentController {
+    private final ComCommentService comCommentService ;
+
+    @PostMapping
+    @Operation(summary = "대댓글 작성")
+    public ResponseEntity<?> create(@RequestBody ComCommentCreateRequest request) {
```

Comment:
```suggestion
    public ResponseEntity<?> create(@Valid @RequestBody ComCommentCreateRequest request) {
```

`@Valid`가 빠져 있어요. 유효성 검사가 되지 않으니 주의해 주세요!!